### PR TITLE
feat: Deploy SpokePoolVerifier that uses depositV3

### PR DESCRIPTION
**Pull request by nicholaspai**

### CODE REVIEW

**Review by james-a-morris — COMMENTED** [review]
We would need to redeploy the FE with this updated deployments.json but this looks great

**Comment by nicholaspai** [comment]
> We would need to redeploy the FE with this updated deployments.json but this looks great

yep exactly

**Review by dohaki — APPROVED** [review]
LGTM

File: deployments/deployments.json
Comment (pxrl) [inline deployments/deployments.json:1]: @nicholaspai I noticed that the mainnet deployment wasn't updated as part of this commit, and it still references the pre-v3 version. Was that intentional?

Comment (nicholaspai) [reply deployments/deployments.json:1]: ah yes i forgot to deploy

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**